integration/proxy/proxy_test.go

```diff
@@ -307,12 +307,13 @@ func TestALPNSNIProxyKube(t *testing.T) {
 	username := helpers.MustGetCurrentUser(t).Username
 	kubeRoleSpec := types.RoleSpecV6{
 		Allow: types.RoleConditions{
-			Logins:     []string{username},
-			KubeGroups: []string{kube.TestImpersonationGroup},
-			KubeUsers:  []string{k8User},
+			Logins:           []string{username},
+			KubernetesLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
+			KubeGroups:       []string{kube.TestImpersonationGroup},
+			KubeUsers:        []string{k8User},
```

Comment:
Same as above. 
Currently, we do not reject the request until #19573 is merged but it's better to add it.